README.md

```diff
@@ -64,6 +64,10 @@ The versioning convention of ECMAScript has been the subject of hot debates. We
 
 [Express](http://expressjs.com/en/index.html) is a fast, unopinionated, minimalist web framework for [Node.js](#nodejs). Express provides a thin layer of fundamental web application features, without obscuring Node.js features that developers already know and like. The myriad of HTTP utility methods and middleware provided by Express makes creating a robust API quick and easy.
 
+## Flux
+
+[Flux](https://facebook.github.io/flux/) is an application structure that is developed and used at Facebook to complement Reacts one-way data flow. With Flux application state and logic are contained in stores.
```

Comment:
Making *React* a link to `#react` would be nice. :)
Also *React’s*, not *Reacts*.
Missing comma after *With Flux*.